numpy/core/fromnumeric.py

```diff
@@ -1875,25 +1895,19 @@ def sum(a, axis=None, dtype=None, out=None, keepdims=np._NoValue):
     -128
 
     """
-    kwargs = {}
-    if keepdims is not np._NoValue:
-        kwargs['keepdims'] = keepdims
+    # Numpy 1.15, 2018-02-25
```

Comment:
This comment normally goes immediately above the warning.

You should add a test for this deprecation in `core/tests/test_deprecations.py` too, along with the same comment with date and version